tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+minversion = 3.7.0
+envlist =
+    clear-coverage
+    quality
+    py35
+    py36
+    py37
+    py38
+    compute-coverage
+skip_missing_interpreters = True
+
+[testenv]
+depends =
+    clear-coverage
+# passenv =
+setenv =
+    PYTHONPATH = {toxinidir}/test_cfg
+    DJANGO_SETTINGS_MODULE = settings
+skip_install =
+    coverage: True
+deps =
+    coverage
+commands =
+    coverage run --parallel-mode --source=git_md_page --branch -m django test git_md_page
+
+[testenv:clear-coverage]
+depends =
+commands =
+    coverage erase
+
+[testenv:compute-coverage]
+depends =
+    py{35,36,37,38}
+commands =
+    coverage combine
+    coverage report --include=*/tests/* --fail-under=100 --show-missing
+    coverage report --omit=*/tests/*
+
+[testenv:quality]
+depends =
+extras = quality
+# Do not fail on first error, but run all the checks
+ignore_errors = True
+commands =
+    isort --recursive --check-only --diff git_md_page
+    flake8 --format=pylint --show-source git_md_page
+    pydocstyle git_md_page
```

Comment:
Chybí black :-) Ten ale nefunguje pod py35, takže bude potřeba udělat toto:

```
deps = 
  !py35: black
set_env =
  py35: BLACK=true
  !py35: BLACK=black
commands:
  ${BLACK} --check --diff --fast ...
```

Reply:
Black added, but not in the mentioned way. This method was not working, I took an inspiration (in the meaning of Ctrl, C and V) in the `niamia` project. Black has been added to the tox and `pyproject.toml` configuration file has been also created.

Reply:
I hoped that we could get rid of that hack from niakampf/niamia. Maybe later when we drop Pythin 3.5.

Reply:
The problem was, that `${BLACK}` was not a known variable or what... Even if I gave it to `setenv` section instead of `set_env` and even if I tried the trick with `{[testenv]setenv}`. Maybe I was just doing something wrong...

Reply:
Resolved.